.mailmap

```diff
@@ -258,7 +259,7 @@ Min Chen <minchen@ubuntukylin.com>
 MingXin Liu <mingxin.liu@kylin-cloud.com> <mingxinliu@ubuntukylin.com>
 Mingyue Zhao <zhao.mingyue@h3c.com>
 Mykola Golub <mgolub@mirantis.com> <mgolub@zhuzha.mirantis.lviv.net>
-Nathan Cutler <ncutler@suse.com> 
```

Comment:
You need to keep 
<pre>
Nathan Cutler &lt;ncutler@suse.com> 
</pre>
and add
<pre>
Nathan Cutler &lt;ncutler@suse.com> &lt;nculter@suse.com>
</pre>
after it.